src/olympia/files/helpers.py

```diff
@@ -60,6 +64,32 @@ def file_tree(files, selected):
     return jinja2.Markup('\n'.join(output))
 
 
+def extract_file(viewer, **kw):
+    # This message is for end users so they'll see a nice error.
+    msg = Message('file-viewer:%s' % viewer)
+    msg.delete()
+    task_log.debug('Unzipping %s for file viewer.' % viewer)
+
+    try:
+        locked = viewer.extract()
+
+        if locked:
+            info_msg = _(
+                'File viewer is locked, extraction for %s could be '
+                'in progress. Please try again in approximately 5 minutes.'
+                % viewer)
+            msg.save(info_msg)
+    except Exception, err:
+        if settings.DEBUG:
+            msg.save(_('There was an error accessing file %s. %s.') %
```

Comment:
It looks like this is creating two strings to be translated though. One with `%s.` at the end and another, that's only used with `settings.DEBUG`, that has `%s. %s.`.

So I think both strings would get extracted but we don't really need to translate the two param version since it's only for development.